src/wagtail_personalisation/__init__.py

```diff
@@ -1 +1 @@
-# default_app_config = 'wagtail_personalisation.config.WagtailPersonalisationConfig'
+default_app_config = 'wagtail_personalisation.config.WagtailPersonalisationConfig'
```

Comment:
@victoriachan I must have left this commented out during testing. Can you see if it fixes the issue as I wasn't seeing the console message. Thanks